cloudcannon.config.yml

```diff
@@ -23,49 +23,14 @@ collections_config:
     _enabled_editors:
       - content
       - visual
-    filter: 
-      base: strict
-  Navigation:
-    output: true
-    path: data
-    disable_add: true
-    disable_add_folder: true
-    icon: table_rows
-    filter:
-       base: none
-       include: 
-        - nav.yml
-        - footer.yml
-  Theme_Palette:
-    output: true
-    path: data
-    disable_add: true
-    disable_add_folder: true
-    icon: color_lens
-    filter:
-       base: none
-       include: 
-        - theme.yml
-  banner:
-    output: true
+
+  data:
+    output: false
```

Comment:
@jonatahb Should this be `disable_url: true`, just to fit the new way of doing things?

Reply:
Fixed.